cpp/src/cider/common/hashtable/HashTable.h

```diff
@@ -383,7 +383,7 @@ struct HashTableFixedGrower {
   size_t next(size_t pos) const { return pos + 1; }
   bool overflow(size_t /*elems*/) const { return false; }
 
-  void increaseSize() { UNREACHABLE(); }
+  void increaseSize() {}

```

Comment:
can remove this method?

Reply:
Of course, it can be removed in next pr or maybe it's also reasonable if we keep them for unified apis of all `Grower`s.